src/CoreBundle/Bridge/Chameleon/Migration/Service/Migrator63.php

```diff
@@ -0,0 +1,412 @@
+            'flag_blue.png' => 'far fa-flag',
+            'tag_blue_edit.png' => 'fas fa-tags',
+            'office-document.png' => 'fas fa-file-alt',
+            'knewsticker.png' => 'fas fa-bullhorn',
+            'kcmdf.png' => 'fas fa-object-ungroup',
+            'news.png' => 'far fa-newspaper',
+            'weather-few-clouds.png' => 'fas fa-cloud-sun',
+            'thumbnail.png' => 'fas fa-images',
+        );
+    }
+
+    /**
+     * Returns map all table icons with new font icons based on the file names.
+     *
+     * @return array
+     */
+    public function getIconMapping(): array
+    {
+        return $this->iconMapping;
+    }
+
+    /**
+     * Returns mapping of old main category groups to new sidebar menu groups.
+     */
+    private function setMainCategoryMapping(): void
+    {
+        $this->mainCategoryMapping = array(
+            'system_website' => 'contents',
+            'shop_article' => 'products',
+            'shop_orders' => 'orders',
+            'customer_shop-einstellungen' => 'system',
+            'system_newsletter' => 'communication',
+            'shop_user' => 'externalusers',
+            'system_search' => 'search',
+            'system_individual_lists_settings' => 'system',
+            'shop_donation_vouchers' => 'discounts',
+            'system_portal_settings' => 'system',
+            'system_admin' => 'system',
+            'system_user_management' => 'internalusers',
+            'system_logs' => 'logs',
+            'shop_article_list_filter' => 'productlists',
+            'system_press' => 'contents',
+            'shop_discount_and_vouchers' => 'discounts',
+            'shop_order_process_settings' => 'checkout',
+            'routing' => 'routing',
+            'system_community' => 'externalusers',
+        );
+    }
+
+    public function addMainCategoryMapping(array $mainCategoryMapping): void
+    {
+        $this->mainCategoryMapping = \array_merge($mainCategoryMapping, $this->mainCategoryMapping);
+    }
+
+    /**
+     * Returns mapping of old main category groups to new sidebar menu groups.
+     */
+    public function getMainCategoryMapping(): array
+    {
+        return $this->mainCategoryMapping;
+    }
+
+    public function migrateUnhandledTableMenuItems(): void
+    {
+        // get all main categories
+        $statement = $this->databaseConnection->executeQuery('SELECT * FROM `cms_content_box` ORDER BY `name` ASC');
+
+        $mainCategoryMapping = $this->getMainCategoryMapping();
+
+        while (false !== $row = $statement->fetch()) {
+            if (isset($mainCategoryMapping[$row['system_name']])) {
+                $sidebarCategorySystemName = $this->mainCategoryMapping[$row['system_name']];
+                $query = 'SELECT * FROM `cms_menu_category` WHERE `system_name` = :systemName';
+                $newMenuGroupId = $this->databaseConnection->fetchColumn($query, ['systemName' => $sidebarCategorySystemName]);
```

Comment:
We should take care of clean wording throughout the PR: In the classic main menu the term "content box" is used, the new main menu uses "category" - we should use those terms instead of inventing the "group" term.

Reply:
renamed group everywhere in variables and method names